bitnami/rabbitmq/README.md

```diff
@@ -263,43 +263,35 @@ extraConfiguration: |
   log.console.formatter = json
 ```
 
-### Recover the cluster from complete shutdown
+## How to Avoid Deadlocked Deployemnts After a Cluster-Wide Restart
 
-> IMPORTANT: Some of these procedures can lead to data loss. Always make a backup beforehand.
+RabbitMQ nodes assume their peers come back online within five minutes (by default). With the `OrderedReady` pod management policy is used
```

Comment:
should this be "*When* the `OrderedReady` pod ..."?

Reply:
Thanks, corrected.